en/docs/learn/fraud-detection.md

```diff
@@ -1,20 +1,28 @@
-# Fraud Detection, Risk Based Authentication and Identity Verification
+# Integrating with Fraud Detection, Risk Based Authentication, Identity Verification and Business Intelligence Systems
 
 ## Overview
 
 ### Fraud Detection
 
 Fraud detection is a set of activities undertaken to prevent information from being acquired under false pretenses.
 Fraud detection can be applied to many industries such as banking and insurance. In banking industry, fraud may include
-forging checks or credit card thefts. When it comes to the identity domain, fraudulent activities include identity
-theft and system intrusions. A system intrusion is an intentional attempt to access/modify unauthorized information
-in the system and convert the system to an unreliable state through information leakage or denial-of-service attacks.
+forging checks or credit card thefts. In e-commerce and retail industry, fraudulent activities mainly circles around
+fraud in purchases which can be any type illegal or false transactions. When it comes to enterprise systems,
+fraudulent activities include identity theft and system intrusions. A system intrusion is an intentional attempt to
+access/modify unauthorized information in the system and convert the system to an unreliable state through
+information leakage or denial-of-service attacks.
 
 Fraud detection involves monitoring the behavior of users in order to estimate anomalous behavior. These behaviors
 can have multiple facets including fraud, intrusion, account defaulting, and delinquency. Presently, fraud detection
 has been implemented by utilizing various methods such as data mining, statistics, and artificial intelligence. All
 of these methods analyze patterns and data about users to identify and prevent suspicious behavior.
 
+In terms of specific industries, the methods and processes used for fraud detection can differ. In Enterprise Fraud
+Management(EFM) systems fraud detection involves real-time screening of transaction activities across users, accounts,
+processes and channels. In E-Commerce and Retail Fraud Management (ERFM) systems, also occupies real-time data
```

Comment:
processes, and channels. 

E-Commerce and Retail Fraud Management (ERFM) systems occupy real-time data...